elements.html

```diff
@@ -2,52 +2,69 @@
 title: Elements
 layout: default
 permalink: /elements/
+subnav:
+  buttons-links: Buttons & Links
+  paragraphs: Paragraph Text
 ---
 
-<h2 class="s-h2">Buttons & Links</h2>
-<p>Many of our sites have links that appear to be buttons. This is acceptable from a design standpoint, but it is important that the HTML correctly communicates the intent of the element to the browser. Start by understanding the important difference between a button and a link:</p>
+<a id="buttons-links"></a>
+<h2 class="s-h2">Buttons & Links</h2></a>
```

Comment:
Is there a specific reason for using a separate `<a>` tag here? Could we just add the `id` to `h2`?

```suggestion
<h2 id="buttons-links" class="s-h2">Buttons & Links</h2></a>
```